deps/openssl/config/archs/BSD-x86/asm/crypto/bf/bf-586.s

```diff
@@ -11,15 +11,15 @@ L_BF_encrypt_begin:
 	movl	16(%esp),%ebp
 	pushl	%esi
 	pushl	%edi
-	# Load the 2 words
+	# Load the 2 words 
```

Comment:
Can these be undone?

Reply:
Every change in the second commit was auto generated. I'd prefer to leave them

Reply:
What's the reason for the whitespace addition? Is it the use of an older Perl version or ?

Reply:
I don't know.